brightlayer-ui/react-showcase-demo
# feature/blui-3123-firebase-hosting config

Fixes # .


#### Changes proposed in this Pull Request:
- add firebase workflows: firebase-hosting-pull-request and firebase-hosting-merge
- firebase-hosting-pull-request yml contains the set expire 2 days. (dev workflow site to test)
- firebase-hosting-merge yml is set to master branch. Workflow kicks off on master merge then deploys it
- 


#### Screenshots:


#### To Test:
- commit a basic change to this PR. The action workflow kicks off and updates the current temp deployment
after merge happens, navigate to the actions tab in this repo. You will see the master deploy workflow run.
